Zend/tests/switch/001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Basic switch expression functionality test
+--FILE--
+<?php
+
+function wordify($x) {
+    return $x switch {
```

Comment:
Why not `return switch ($x) {`?

Reply:
https://externals.io/message/109289

Reply:
I'm open to suggestions. There are some parser hacks to make it unambiguous (the expr version can't be at the start of the statement) but that would make it context sensitive which is something that has always been rejected in the past.

Reply:
Thanks for the link!